libs/elements/banners/src/lib/components/elewa-image-and-details-banner/elewa-image-and-details-banner.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+
+import { ImageAndDetails } from '@elewa-website/models/schema/ui/banners';
+
+@Component({
+  selector: 'elewa-website-elewa-image-and-details-banner',
```

Comment:
rename this selector to 'elewa-image-and-details-banner'

Reply:
alright